packages/crayons-extended/platform-ui/src/components/filter/readme.md

```diff
@@ -0,0 +1,141 @@
+# fw-filter
+
+A Dynamic filter component that will generate a filter form based on the conditionSchema and filters.
```

Comment:
Please add a working demo for all the platform-ui components.

Reply:
Added examples